_data/navigation.yml

```diff
@@ -20,5 +20,6 @@
   subpages:
   - url: /outreach
     title: Outreach
-  - url: /sponsors
-    title: Sponsors
+
+- url: /sponsors
```

Comment:
keep formatting as it was

Reply:
I talked with Meredith a little before hand and we felt that it'd be good to have our sponsors page be a little more visible since it'll still fit outside of a drop-down menu. I'm probably not aware of the reason for lumping it in a drop down with Community though. 

What would you think of eliminating the drop-down all together and just give Outreach and Sponsors their own place on the navbar?

Reply:
Oh whoops, I didn't actually realize that's what this was doing, my bad! The only issue I see is that, on mobile, it makes the navbar 2 lines (we need to redesign the mobile site anyways so not a _huge_ deal). How would you feel about moving "Parents" to the community tab?

Reply:
Good idea!

Reply:
Actually, on second thought, how does moving "Leadership" under "About" sound? And then we can just replace "Community" with "Outreach." Parents doesn't seem to quite fit under 'Community' to me

Reply:
That works for me, but move all the dropdowns to the end of the row. I think robots might also fit into that drop-down.

Reply:
👍 I kept the "About" button where it was before for now -- I feel like most people will assume that it's a dropdown. I feel like "Robots" is a rather important page and should be visible at the top-level so I still put "Parents" under "Community" and moved it to the end so that we wouldn't have too many things on the navbar